### PR TITLE
write the install file, re-organize the hardware files

### LINKED ISSUE #18: write the install file

### PR DESCRIPTION
**Pull request by zramsay**
- closes #18, #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Point of Sale
 
 ## Introduction
-This application is two things. First, it is a point of sale system designed for small businesses that have an entry fee and also sell various items. Second, it is an introductory programming application that bridges the gap between a `hello world` example and advanced tutorials.
+This application is two things. First, it is a point of sale system designed for small businesses that have an entry fee and also sell various items. Second, it is an introductory/example/demo programming application that bridges the gap between a `hello world` or `TODO` list example and advanced tutorials.
```
Comment (danielhunt) [inline README.md:4]: ;) @ `TODO`

File: hardware.go [hunk 2]
```diff
@@ -8,6 +8,15 @@ import (
 	"strings"
 )
 
+// alert can be a buzzer or a light, or something else ...
+func activateAlert() error {
+	_, err = exec.Command("python", "hardware/alert.py").Output()
```
Comment (danielhunt) [inline hardware.go]: Now that we have a functioning `config`, why not add the types of alerts that are enabled into it, and split them out into multiple files?
That way, we could add `buzz` and `print` alert types, and just iterate through the enabled ones here.

It'd allow for easy extension to other alert types too, like `email` or even `sms` :D

Comment (danielhunt) [reply hardware.go]: And actually, it's more of a `Notification` than an `Alert`, right?

Comment (zramsay) [reply hardware.go]: Nofication does make more sense, thanks. Not a bad idea for config, although that can be part of a seperate feature/pr

Comment (danielhunt) [reply hardware.go]: :+1: @ feature expansion

**Pull request merged**